README.md

```diff
@@ -32,15 +32,15 @@ This plugin is compatible with CSI versions [v1.1.0](https://github.com/containe
 
 ### Kubernetes Compatibility
 
-| GCE PD CSI Driver\Kubernetes Version | 1.10.5 - 1.11 | 1.12 | 1.13 | 1.14 | 1.15+|
-|--------------------------------------|---------------|------|------|------|------|
-| v0.1.x.alpha                         | yes           | no   | no   | no   | no   |
-| v0.2.x (alpha)                       | no            | yes  | no   | no   | no   |
-| v0.3.x (beta)                        | no            | no   | yes  | yes  | yes  |
-| v0.4.x (beta)                        | no            | no   | yes  | yes  | yes  |
-| v0.5.x (beta)                        | no            | no   | no   | yes  | yes  |
-| v0.6.x (beta)                        | no            | no   | no   | yes  | yes  |
-| dev                                  | no            | no   | no   | yes  | yes  |
+| GCE PD CSI Driver\Kubernetes Version | 1.10.5 - 1.11 | 1.12 | 1.13 | 1.14 | 1.15 | 1.16+ |
```

Comment:
The table is starting to get big. Should we start pruning unsupported driver versions from this?  Or instead of a matrix, for each driver version, just give the range of supported K8s versions in a single column?

Reply:
can address in a follow-up/refactor. lots of moving parts in this PR as-is